cli/src/trusted_base_cli/commands/pay_as_bid_proof.rs

```diff
@@ -54,14 +54,14 @@ pub(crate) fn pay_as_bid_proof(
 	cli: &Cli,
 	trusted_args: &TrustedCli,
 	arg_who: &str,
-	orders_file: &str,
+	orders_string: &str,
```

Comment:
I'm getting this error with the new changes:
```
 self.orders_string,
   |                 ^^^^^^^^^^^^^^^^^^ move occurs because `self.orders_string` has type `std::string::String`, which does not implement the `Copy` trait
   ```

Reply:
Ah ok, I did not see this detail. Then you can pass the argument with: `orders_string.clone()`.

-> copy: for statically sized data types. Copies the data on the stack, happens implicitly.
-> clone: for dynamically sized data types. Needs to be done explicitly with `clone`.

Reply:
done